libraries/vue/src/components/ChatUi.vue

```diff
@@ -0,0 +1,60 @@
+import {connectApi, loadChatUi} from "chatkitty";
+
+<script setup lang="ts">
+
+import {connectApi, loadChatUi} from "chatkitty";
+
+const {apiKey, username, widgetId} = defineProps<{ apiKey: string, username: string, widgetId: string }>()
+
+const initializeChat = async () => {
+  const connection = await connectApi({
+    apiKey: apiKey,
+    username: username
+  });
+
+  const {user, unreadChannelsCount, notifications, updateUser} = connection;
+  console.log("Connected as user:", user.value);
+
+  user.watch((user) => console.log("User:", user));
+  unreadChannelsCount.watch((count) => console.log("Unread channels count:", count));
+  notifications.watch((notification) => console.log("Received notification:", notification));
+
+  loadChatUi(
+      {
+        widgetId: widgetId,
+        container: {height: "100%"},
+        audio: {enabled: true},
+        components: {
+          chat: (context) => ({
+            menuActions: [],
+            onMounted: () => {
+              console.log("Chat UI mounted with context:", context);
```

Comment:
We can take this time to remove the miscellaneous console log statements and `updateUser` example call - these are leftovers from tests and examples from the original demo.

Reply:
Eventually, we'll add event hooks, etc. for folks to handle events like on mounted from chat-ui